### PR TITLE
Fix consul curl issue on Travis

### PR DESCRIPTION
**Pull request by mhamann**
Testing a possible fix for the curl issue.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: go
+dist: trusty
```
Comment (janisz) [inline .travis.yml]: I think we need 
```yaml
sudo: false
dist: trusty
```

Comment (mhamann) [inline .travis.yml]: The build made it much further, but hit another error.

Comment (janisz) [inline .travis.yml]: It passed for me after adding sudo line  https://travis-ci.org/allegro/marathon-consul/builds/195645241

**Comment by janisz** [comment]
@mhamann Would you like to update this PR?

**Comment by mhamann** [comment]
Will do a bit later today. Thanks!

**Comment by janisz** [comment]
For some reason build hangs. Maybe we should update consul to 7.3

**Comment by mhamann** [comment]
The build hang seems to be intermittent. I did bump Consul to 0.7.3, but does not seem to have alleviated the problem.

Something is flaky...

**Comment by janisz** [comment]
@mhamann Thanks!

**Pull request merged**